Gruntfile.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(grunt) {
+   grunt.initConfig({
+       jshint: {
+          files: ['Gruntfile.js', 'test/**/*.js', 'lib/*.js', '*.js']
```

Comment:
If you use the /**/*.js syntax you should also apply it to lib/. Instead of specifying which subdirectories you care about, you are saying to check all subdirectories under test/ or lib/.